content/docs/tutorial-part-1-contributor.rst

```diff
@@ -109,7 +109,7 @@ Like our git repo, Radicle stores our issues and patches locally and distributes
 
   $ rad issue list
   state    #   title                         author    updated
-  open     0   Patch proposal not accepted   jane      2019-03-04T16:41:47Z
+  open     0   Patch proposal not accepted   jane      2019-03-04T16:41:47Z *
 
```

Comment:
Am I understanding this correctly that it's just added to the `updated` column and not it's own `unread` column?

Reply:
Indeed, feel free to propose something else though.